src/routes/contact-us/+page.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+	import fb_logo from './Facebook_Logo_Primary.png';
+	import ig_logo from './Instagram_Glyph_Gradient.png';
+</script>
+
+<h1>Contact Us</h1>
+
+<p>
+	Email Address: <a href="mailto:kendoinfo@umich.edu"> kendoinfo@umich.edu</a>
+</p>
+
+<h2>Find us on:</h2>
+
+<figure>
+	<a href="https://www.facebook.com/groups/1038072369705184">
+		<img class="social-media" src={fb_logo} alt="Facebook logo" />
+	</a>
+	<a href="https://www.instagram.com/kendoumich/">
+		<img class="social-media" src={ig_logo} alt="Instagram logo" />
```

Comment:
I don't think the alt text flows well here.  Try saying this out loud: "Find us on Instagram logo" it should probably just be "Instagram". You could also consider using title instead of alt

Reply:
I don't think I should skip using alt for an image. Svelte gives me a warning message. What do you think about using both? That way people can have informational text on hover too?

Reply:
No definitely don't do both. Screen readers should read the title if there is one. I don't think svelte should be giving you an error if you have a title but not a alt, let me know if it does.

Reply:
In the first link the author is finding that the title attributes is ignored when it is placed on elements that already have text like `<a>`. He thinks that he can get away with having links that just say "click here" (never do this BTW) if he puts a title on them to explain where "here" is. He finds out that this doesn't work because screen readers typically ignore titles on elements with text.

The A11y website says that title should only be used in two cases but one of them is "For providing a label when a text label would be redundant" which applies to our situation here. It confused me at first that they said titles should be the same as alt text but then went on to explain an algorithm that will correctly use the title if there is no alt text but I think what they mean is that if you have a title and an alt then they should be the same.

You should test it with Windows narrator and Android TalkBack just to be safe.

Reply:
Thank you, I understand now! I tested it with both Windows narrator and Android TalkBack. Both worked with the title attribute without alt text for the images. There's also evidence from 2023 that[ it's supported in more browsers/screen readers](https://www.powermapper.com/tests/screen-readers/labelling/a-img-title/). I'm a bit worried future developers on the website will be concerned about the warnings, but that's why we have a code review system.
Resolved in [313846a](https://github.com/kendoclub-at-umich/website/pull/32/commits/313846aae2e5652f06111a4d1a68a21122224dfc)

Reply:
> I'm a bit worried future developers on the website will be concerned about the warnings

What warnings?

Reply:
Alright nevermind ESlint doesn't like not having alt text for images. I think I'll just switch back from title to alt text.

Reply:
These warnings 
![image](https://github.com/kendoclub-at-umich/website/assets/75005450/cce58f55-4e24-4e84-b78e-a77753ab8e7f)

Reply:
We should look into those warnings and see if there is a way to disable them when title is present or if they have a good reason it should be required even when title is there. Or may if there is a way to assert that title and alt are not different.

Reply:
After conversation, we decided to relocate/move some of the social media. Resolved in [5e58e97](https://github.com/kendoclub-at-umich/website/pull/32/commits/5e58e970d6d50441610fd35db7b9caf3c7f235e7)